### DISCUSSION TITLE
Socket Viewer: Unable to view Map viewer after SLAM run using socket viewer

### DISCUSSION THREAD

**Discussion by Udaykiran87** [stella-cv/stella_vslam#91] [Q&A] [answered]
Hello,

I followed below url to run sample tutorial of OpenVSLAM.
https://openvslam-community.readthedocs.io/en/latest/simple_tutorial.html#chapter-simple-tutorial

Note:
I used Instructions for SocketViewer for docker run.

Problem:
Under Tracking and Mapping section, I executed below command to run SLAM.
$ ./run_video_slam \
    -v /path/to/orb_vocab/orb_vocab.fbow \
    -c /path/to/openvslam/example/aist/equirectangular.yaml \
    -m /path/to/aist_living_lab_1/video.mp4 \
    --frame-skip 3 \
    --map-db aist_living_lab_1_map.msg

I am able to view 'Frame Viewer' in web browser: http://localhost:3001/
But i am unable to watch  'Map viewer' window.

Am i missing some thing here?

Any help would be highly appreciated.

Thanks and Regards,
Udaykiran Patnaik.

**Comment by ymd-stella** [comment, marked as answer]
Please post a screenshot.

**Reply by Udaykiran87** [reply]
Hello ymd-stella,

I am sorry, I misunderstood. I can see both view.

Thanks and Regards,
Udaykiran Patnaik.
